lib/presenters/slack/full_message.rb

```diff
@@ -1,29 +1,34 @@
 module Presenters
   module Slack
     class FullMessage
-      def execute(pull_requests:, team_name:)
-        "##{team_name} You have #{pull_requests.count} Dependabot PRs open on the following apps:
+      def execute(pull_requests:)
+        "##{team_name(pull_requests)} You have #{pull_requests_count(pull_requests)} Dependabot PRs open on the following apps:
 
-#{body(grouped_by_application(pull_requests)).join("\n")}
+#{body(pull_requests).join("\n")}
 
 Feedback: https://trello.com/b/jQrIfH9A/dependabot-developer-feedback"
       end
 
     private
 
-      def grouped_by_application(prs)
-        prs.group_by { |pr| pr.fetch(:application_name) }
-      end
-
       def body(pull_requests)
-        pull_requests.map do |application_name, _|
+        pull_requests.fetch(:applications).map do |application|
+          application_name = application.fetch(:application_name)
           application_name + ' ' + url(application_name)
         end
       end
 
+      def team_name(pull_requests)
+        pull_requests.fetch(:team_name)
+      end
+
       def url(application_name)
         "https://github.com/alphagov/#{application_name}/pulls/app/dependabot"
       end
+
+      def pull_requests_count(pull_requests)
+        pull_requests.fetch(:applications).reduce(0) { |acc, pr| acc + pr.fetch(:pull_request_count) }
```

Comment:
Wonder if this should happen in UseCases::GroupApplicationsByTeam

Add pull_requests_count